server/js/topics.js

```diff
@@ -0,0 +1,77 @@
+class DAS {
+  static data = 'data';
+  static start = 'start';
+  static stop = 'stop';
+}
+
+class BOOST {
```

Comment:
As before, could add a `name` static property (which shadows the existing `Function.name` property).